nat-lab/tests/utils/vm/mac_vm_util.py

```diff
@@ -14,14 +15,18 @@
 
 VM_TCLI_DIR = LIBTELIO_BINARY_PATH_MAC_VM
 VM_UNIFFI_DIR = UNIFFI_PATH_MAC_VM
+CONNECTION_TIMEOUT = 10
```

Comment:
are we sure that 10 seconds will be enough? also why this is really needed, at some point `asyncssh.connect` will timeout itself. This is kinda handled by system's default TCP connect timeout or can be configured by setting `connect_timeout` at `SSHClientConnectionOptions`

Reply:
I've decided to restore this to original and rely on the builtin tcp timeout.